addons/website/models/website.py

```diff
@@ -857,6 +857,10 @@ def search_pages(self, needle=None, limit=None):
                 break
         return res
 
+    def get_suggested_controllers(self):
+        suggested_controllers = []
```

Comment:
add homepage here.
Usefull and probably help to easily override.

Because now when I read it, it is a lsit of what ? string url ? tuple(translated ? name, url), ...
Please document it with a docstring